intelmq/bots/parsers/bitsight/parser.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""
+Bitsight Alert Stream parser
+
+_ts                                     Timestamp in Unix Time                  => time.source
+env.remote_addr                         IP that accessed the sinkhole           => source.ip
+env.remote_port                         Port that accessed the sinkhole         => source.port
+env.server_addr                         Sinkhole IP                             => destination.ip
+env.server_port                         Sinkhole port                           => destination.port
+env.server_name                         Domain accessed by IP                   => destination.fqdn
+env.request_method                      Method used by IP                       => extra.method
+trojanfamily                            Malware name                            => malware.name
+_geo_env_remote_addr.country_name       Country location of the IP              => source.geolocation.country
+"""
+
+import json
+import sys
+from intelmq.lib import utils
+from intelmq.lib.bot import Bot
+from intelmq.lib.message import Event
+from intelmq.lib.harmonization import DateTime
+from intelmq.lib.exceptions import InvalidValue
+
+
+class BitsightParserBot(Bot):
+
+    def process(self):
+        report = self.receive_message()
+        if report is None or not report.contains('raw'):
+            self.acknowledge_message()
+            return
+        raw_report = json.loads(utils.base64_decode(report.get('raw')))
+        extra = {}
+        event = Event(report)
+        event.add("raw", report.get('raw'))
```

Comment:
Either add `raw_report` or add the parameter `sanitize=False` (preferred). Otherwise it will be encoded two times.
(Spotted this when inspecting the tests :) )